File: backends/php-laravel/app/Infrastructure/Persistence/Eloquent/Models/Invitation.php
<?php
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
namespace App\Infrastructure\Persistence\Eloquent\Models;

use App\Support\Models\AppMasterModel;
use Database\Factories\InvitationFactory;

/**
 * 招待Modelクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 * @package App\Infrastructure\Persistence\Eloquent\Models
 */
class Invitation extends AppMasterModel
{
    //

    /**
     * {@inheritdoc}
     */
    protected static function newFactory()
    {
        return InvitationFactory::new();
    }
}

File: backends/ruby-rails/app/infrastructure/model/invitation.rb
# frozen_string_literal: true
#
# 招待モデルを定義するインフラストラクチャモジュール。
#
# @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>

module Infrastructure
  module Model
    # invitations テーブルに対応する ActiveRecord モデルです。
    class Invitation < ActiveRecord::Base
      self.table_name = "invitations"
    end
  end
end
